api/tests/integration/infrastructure/repositories/target-profile-repository_test.js

```diff
@@ -629,4 +629,58 @@ describe('Integration | Repository | Target-profile', () => {
     });
   });
 
+  describe('#findOrganizationIds', () => {
+    let targetProfileId;
+    const expectedOrganizationIds = [];
+
+    beforeEach(() => {
+      targetProfileId = databaseBuilder.factory.buildTargetProfile().id;
+      _.times(2, () => {
+        const organizationId = databaseBuilder.factory.buildOrganization().id;
+        expectedOrganizationIds.push(organizationId);
+        databaseBuilder.factory.buildTargetProfileShare({ organizationId, targetProfileId });
+      });
+      return databaseBuilder.commit();
```

Comment:
**suggestion (non-blocking) **: On pas besoin d'avoir deux organisations associées dans chaque test
Pour les tests suivants une seule organisation suffit :
- `should not include an organization that is not attach to target profile`
- `when no organization is linked to the target profile`
- `when target profile does not exist`

C'est ok pour de le laisser comme ça pour moi, mais ça rallonge les tests (plus de données) et ça lie tous les tests (jeu de données commun).
Si ce qu'on souhaite c'est mutualiser du code  on pourrait combiner des tests par exemple : `should return an Array of Organization ids` et `should not include an organization that is not attach to target profile`.